command/ibft/ibft_init.go

```diff
@@ -72,20 +80,30 @@ func (p *IbftInit) Run(args []string) int {
 		return 1
 	}
 
-	if err := minimal.SetupDataDir(dataDir, []string{"consensus", "libp2p"}); err != nil {
+	subDirectories := []string{"consensus", "libp2p"}
```

Comment:
Please refactor these to constants, no need for a slice. It would be much more readable below to reference LIBP2P_DIR than subDirectories[1].

Reply:
Agreed 👍 

Fixed 👍 
1f7127d